src/redux/actions/resetPasswordAction.js

```diff
@@ -0,0 +1,22 @@
+import {RESET_ACTION,SEND_RESET_EMAIL_ACTION,SEND_RESET_EMAIL_IN_PROGRESS,RESET_IN_PROGRESS} from './actionTypes';
+import axios from 'axios';
+const sendResetEmailAPILink='https://warriorz-staging.herokuapp.com/api/v1/password/forgot';
```

Comment:
For the sake of security, I think these two routes may be better if they were obtained from the environmental variables in the `.env`

Reply:
Also, try to resolve the conflicts and make sure everything is good.